elements/pfe-autocomplete/src/pfe-autocomplete.js

```diff
@@ -431,9 +466,7 @@ class PfeSearchDroplist extends PFElement {
 
     let options = this.data;
 
-    this._ariaAnnounce.innerHTML = `There are ${
-      options.length
-    } suggestions. Use the up and down arrows to browse.`;
+    this._ariaAnnounce.innerHTML = `There are ${options.length} suggestions. Use the up and down arrows to browse.`;
```

Comment:
Need to get this content from the light DOM because otherwise it can't be translated.

Reply:
I've added an `aria-announce-template` attribute that allows a user to add their own string or translated string that will be used in the aria-live section of the component.